### PR TITLE
Feature/#82 introduce melos and dvmx

### LINKED ISSUE #82: [Improve]: Standardize Dart and Dependency Versions for CLI Apps Under 'tools'
### Describe

- No need to align the Dart version for 'yumemi_lints' as it's not feasible
- Manage the 'tools' directory using 'melos'
- Utilize DVM for managing Dart versions

### Screenshots

N/A

### Environment

- Commit SHA: c0045ee1f4416634ec74acecd58510f68687c5bc

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Yamasaki-pan961**
## Issue

- close #82 

## Overview (Required)

- Introduce dvmx
- Introduce melos
- Add GET_STARTED.md
- Standardize dependency version
- Refactor and Update launch.json

## Links

In some cases, melos commands and scripts do not work because melos and Version Manage are not compatible.
- https://github.com/invertase/melos/issues/616

### CODE REVIEW

**Review by blendthink — COMMENTED** [review]
@Yamasaki-pan961 
Thank you!

Could you please rewrite the GitHub Actions workflow using the following GitHub Action?
https://github.com/blendfactory/dvm-config-action

**Comment by Yamasaki-pan961** [comment]
@blendthink 
> Could you please rewrite the GitHub Actions workflow using the following GitHub Action?
> https://github.com/blendfactory/dvm-config-action

Changed to use it!!
9a00902494ab359b59508918b73f255649a2a695

File: docs/contributing/GET_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Please install the necessary tools before making a contribution.
+
+## 1. Install DVMx
+[dvmx] is Dart version management tool.
+1. Install dvmx globally. ( [install dvm] )
+2. Execute the following command in the project root.
+```bash
+dvm use
+```
+1. Run the command `dvm dart --version` to check the version.
```
[the remark this answers is not in the document]
Comment (Yamasaki-pan961) [reply docs/contributing/GET_STARTED.md]: fixed: 8100782a30228eeefc4a1995f96abbdf51dbd24f

**Review by blendthink — APPROVED** [review]
LGTM！！

**Pull request merged**